docs/pubsub-api.rst

```diff
@@ -0,0 +1,255 @@
+``gcloud.pubsub`` API
+=====================
+
+Connection / Authorization
+--------------------------
+
+- Inferred defaults used to create connection if none configured explicitly:
+
+  - credentials (derived from GAE / GCE environ if present).
+
+  - ``project_id`` (derived from GAE / GCE environ if present).
+
+  - ``scopes``
+
+
+Manage topics for a project
+---------------------------
+
+Create a new topic for the default project:
+
+.. doctest::
+
+   >>> from gcloud import pubsub
+   >>> topic = pubsub.create_topic("topic_name")
+   >>> topic.name
+   'topic_name'
+
+Create a new topic for an explicit project:
+
+.. doctest::
+
+   >>> from gcloud import pubsub
+   >>> topic = pubsub.create_topic("topic_name", project_id='my.project')
+   >>> topic.name
+   'topic_name'
+
+Fetch an extant topic for the default project:
```

Comment:
The connotation of [extant](http://www.merriam-webster.com/dictionary/extant) is that a thing (document, for instance), which might have been destroyed, still exists.  Given that topics might be destroyed between the time they were initially created and the time the `get_topic()` call is made, that connotation is precisely what I meant.